src/XBDMCommunicator/XDevkitSnub/SnubException.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace XBDMCommunicator.XDevkitSnub
+{
+	public static class SnubException
+	{
+		public static string Message { get { return "Must be build with XDevkit to run this."; } }
```

Comment:
nit: typo
```suggestion
		public static string Message { get { return "Must be built with XDevkit to run this."; } }
```
Also, maybe this should be more detailed and point people to the XDevkit solution configurations.

Reply:
Also a great idea